scripts/xtensa-build-zephyr.sh

```diff
@@ -58,6 +58,8 @@ clone()
 	mkdir -p "$WEST_TOP"
 	git clone --depth=5 https://github.com/zephyrproject-rtos/zephyr \
 	    "$WEST_TOP"/zephyr
+	git -C "$WEST_TOP"/zephyr --no-pager log --oneline --graph \
+	    --decorate --max-count=20
```

Comment:
good idea, but isn't max-count=20 too many?

Reply:
20 does seem a bit long, however:
 1. The entire build log is (unfortunately) over 2000 line long, so 20 wouldn't make that much difference
 2. It's not even 20 actually. zephyr seems to keep an always linear history, meaning in practice there will never be more than 5 commits listed (confirmed in example above). If there is ever a git merge with 2 parents then there will not more than 10 commits listed.

Think of this max-count as a "safety valve".

Reply:
>zephyr seems to keep an always linear history
This part I don't understand, but I saw the example log. Looking nice with 5 logs.